src/switch/switch.stories.ts

```diff
@@ -10,7 +10,10 @@ storiesOf("Switch", module).addDecorator(
 )
 	.addDecorator(withKnobs)
 	.add("Basic", () => ({
-		template: `<ibm-switch [disabled]="disabled"></ibm-switch>`,
+		template: `
+		<ibm-switch [disabled]="disabled"></ibm-switch>
+		<ibm-switch [disabled]="disabled" [size]="'sm'"></ibm-switch>
```

Comment:
This can be further simplified as:
```suggestion
			<ibm-switch [disabled]="disabled" size="sm"></ibm-switch>
```